src/Behat/Config/Config.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Behat\Config;
+
+final class Config
+{
+    public function __construct(
+        private array $settings = []
+    ) {
+    }
+
+    public function import(string ...$files): self
+    {
+        foreach ($files as $file) {
+            $this->settings['imports'][] = $file;
```

Comment:
I might have missed something, but if the `->import()` method just adds keys/values to the array you are already defining in the file, why wouldn't you just do that inline?

E.g. 
```
return new Config([
  'imports' => [
    __DIR__.'/other-config.php'
  ],
  // Other values 
])
```